### PR TITLE
BMP support

### PR DESCRIPTION
**Pull request by borodean**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  quotes: [2, "single"]
+  linebreak-style: [2, "unix"]
+  semi: [2, "always"]
+  no-cond-assign: [0]
+  space-before-blocks: [2, "always"]
+  space-before-keywords: [2, "always"]
+  space-before-function-paren: [2, "always"]
+env:
+  node: true
+  mocha: true
```
Comment (mgartner) [inline .eslintrc]: Can you remove this file? I've since added eslint configs to the repo.

**Comment by mgartner** [comment]
Looks great, looks like you need to rebase.

Also, Would you mind including a few more BMP files that you find randomly over the web or wherever to be extra sure there are no edge cases?

**Comment by borodean** [comment]
@mgartner I've searched for more image samples and have found that there is an older OS/2 BMP format for which I haven't introduced recongnition. I've added that sample among others and improved the code so it works now.

**Comment by mgartner** [comment]
Looks great thanks for contributing!

**Comment by borodean** [comment]
@mgartner always a pleasure :smile:

**Pull request merged**